sklearn/decomposition/_nmf.py

```diff
@@ -913,6 +913,9 @@ def non_negative_factorization(X, W=None, H=None, n_components=None, *,
 
         - 'custom': use custom matrices W and H
 
+        Ignored if update_H is False, in which case the initial value of W
+        is set automatically.
```

Comment:
This may be clearer:

```
- 'custom': use custom matrices W and H if `update_H=True`. If `update_H=False`, then
  only custom matrix H is used.
```

The `init=='custom'` and `update_H=False` will down the following code path:

https://github.com/scikit-learn/scikit-learn/blob/f27adc51d5dc4213844093c506b55fdcf0daed58/sklearn/decomposition/_nmf.py#L1040-L1050